CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changes
+
+* The `Plugin.__call__` method now allows arbitrary keyword arguments to be
```

Comment:
thanks for these changes @paulromano - for the plugin option to pass arguments to `run` - I suggest to update one of the examples to showcase this option.

Reply:
Good suggestion -- I just updated the usage documentation for PluginOpenMC to show how this works